### ISSUE TITLE
Error: project ID is required

### ISSUE THREAD

**Issue by Eddict** [shawntabrizi/ethereum-balance-graph#1]
tried to test it on https://shawntabrizi.com/ethgraph/ but generates "Error: project ID is required"

**Comment by Chardonneaur** [comment]
yes, that's because you need to fill the js file with the infura id
